guidelines/datanote.md

```diff
@@ -0,0 +1,46 @@
+**_Journal of Cheminformatics_ will only publish research or software that is entirely reproducible by third parties.
+This means that any datasets, software and algorithms that are required to reach the conclusions stated in the
+paper must be provided as supplemental materials, or be otherwise accessible without the need for registration,
+login or agreement with license terms other than Creative Commons licenses for data and text and
+[OSI-approved Open Source Licenses](http://opensource.org/licenses/alphabetical)
+for software. For any software, the source code must be provided.**
+
+A Data Note should describe a curated, CCZero or CC-BY-licensed dataset likely to be of broad utility. The dataset must
+be curated as detailed in this publication and archived for long-term reuse. A Data Note describes the content
+of the data set, the method used to curate the data, and a description of how the dataset deposition complies with
+the [FAIR principles](https://doi.org/10.1162/DINT_R_00024). Unlike with a Database paper, Data Notes focus on the
+data itself and not how the data can be graphically or otherwise explored.
+
+The structure of a Data Notes paper, should include the following components:
+
+* an abstract that does not exceed 350 words
+* an Objective section
+* a Data description, with two subsections called Curation and FAIR-ification
+* a Limitations section
+* followed by the common sections Abbreviations, Declarations, and References
+
+The main section of the article starts with a Objective section explaining where the data came
+from and why it was collected citing a selection of relevant literature, particularly articles describing
+data (re)used in this article, and how this data is or can be used. The Limitations section should
+discuss the limitations of the data to help the reader to assess the reusability. When your
+data is collected from other data, adhere to the [DataCite](https://datacite.org/cite-your-data.html) principles
+(and please use *[cito:usesDataFrom]*, see the Citation Typing Ontology section below).
+
+The Data description should contain a table describing the data files presented in this publication
+in a general paragrah, following by two subsections describing the methodologies used curate the data
```

Comment:
Is this being a little too prescriptive?

Reply:
Intentional, to simplify review, and compatibility with BMC Research Notes.